File: customercenter/src/main/java/cafeteria/external/KakaoServiceImpl.java
package cafeteria.external;

import org.springframework.stereotype.Component;

import lombok.extern.slf4j.Slf4j;

@Slf4j
@Component
public class KakaoServiceImpl implements KakaoService {

	@Override
	public void sendKakao(KakaoMessage message) {
		log.info("\nTo. {}\n{}\n", message.getPhoneNumber(), message.getMessage());
	}

}

File: customercenter/src/main/scala/cafeteria/external/KakaoServiceImpl.scala
package cafeteria.external

import org.springframework.stereotype.Component

@Component
class KakaoServiceImpl extends KakaoService {
  
	override def sendKakao(message :KakaoMessage) {
		logger.info(s"\nTo. ${message.phoneNumber}\n${message.message}\n")
	}
}
